web/src/main/webapp/WEB-INF/pages/candidates.jsp

```diff
@@ -39,9 +39,9 @@
             <div id="new_candidate" class="text-right">
                 <a class="btn btn-success" href="${newcandidate}" id="add_candidate_button"><spring:message code="welcome.candidates.add.button"/></a>
             </div>
-            <div id="candidates_table">
+            <div id="candidates_table" >
                 <div>
-                    <table class="table table-striped table-hover" data-toggle="table" id="candidates" data-url="${candidates}" data-height="500" data-sort-name="name" data-escape="true">
+                     <table class="table table-striped table-hover cursor-pointer" data-toggle="table" id="candidates" data-url="${candidates}" data-height="500" data-sort-name="name" data-escape="true">
```

Comment:
Code formatting issue.

Delete new extra space character in the beginning of the line.